common/reviews/api/node-core-library.api.md

```diff
@@ -650,5 +657,11 @@ export enum TextAttribute {
     Underline = 2
 }
 
+// @public
+export class UuidTypeId {
+    static isInstanceOf(targetObject: unknown, typeUuid: string): boolean;
+    static registerClass(targetClass: new (...args: any[]) => any, typeUuid: string): void;
+    }
```

Comment:
Want to use this in [`FileError`](https://github.com/microsoft/rushstack/blob/master/apps/heft/src/pluginFramework/logging/FileError.ts) as well?

Reply:
Good idea, fixed